webchat/auth/authService.js

```diff
@@ -0,0 +1,206 @@
+(function() {
+    'use strict';
+
+    const app = angular.module("webchat");
+
+    app.service("AuthService", function AuthService($q, $state, $window, UserService, MessageService,
```

Comment:
use Inline array annotation here and for all dependency injectors
https://docs.angularjs.org/guide/di#inline-array-annotation